src/Val.cc

```diff
@@ -721,7 +723,8 @@ void IntervalVal::ValDescribe(ODesc* d) const
 			if ( ! (v >= unit || v <= -unit) )
 				continue;
 
-			double num = static_cast<double>(static_cast<int64_t>(v / unit));
+			double num = v / unit;
+			num = num < 0 ? std::ceil(num) : std::floor(num);
```

Comment:
Is there a reason this isn't a round()?

Reply:
Changing to `round()` makes things like `8.5 days` format like `9.0 days -12.0 hrs`, which is technically still right, but probably not what we want to go with :)